mmv1/products/firebaseapphosting/Backend.yaml

```diff
@@ -0,0 +1,215 @@
+# Copyright 2025 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: Backend
+description: A Backend is the primary resource of App Hosting.
+base_url: projects/{{project}}/locations/{{location}}/backends
+update_mask: true
+self_link: projects/{{project}}/locations/{{location}}/backends/{{backend_id}}
+create_url: projects/{{project}}/locations/{{location}}/backends?backendId={{backend_id}}
+# Always forces deletion because the child resource default `Domain` is always present.
+delete_url: projects/{{project}}/locations/{{location}}/backends/{{backend_id}}?force=true
+update_verb: PATCH
+id_format: projects/{{project}}/locations/{{location}}/backends/{{backend_id}}
+import_format:
+  - "projects/{{project}}/locations/{{location}}/backends/{{backend_id}}"
+  - "{{project}}/{{location}}/{{backend_id}}"
+  - "{{location}}/{{backend_id}}"
+min_version: beta
+examples:
+  - name: firebase_app_hosting_backend_basic
+    primary_resource_id: example
+    min_version: beta
+    vars:
+      location: "us-central1"
+      backend_id: "my-backend"
+      app_id: "1:0000000000:web:674cde32020e16fbce9dbd"
+      display_name: "My Backend"
+      serving_locality: "GLOBAL_ACCESS"
+    test_env_vars:
+      project_id: "PROJECT_NAME"
+    test_vars_overrides:
+      "location": '"us-central1"'
+      "serving_locality": '"GLOBAL_ACCESS"'
+      "app_id": '"1:0000000000:web:674cde32020e16fbce9dbd"'
+  - name: firebase_app_hosting_backend_github
+    primary_resource_id: example
+    min_version: beta
+    vars:
+      location: "us-central1"
+      backend_id: "my-backend-gh"
+      app_id: "firebase:app:id"
+      display_name: "My Backend"
+      serving_locality: "GLOBAL_ACCESS"
+    test_env_vars:
+      project_id: "PROJECT_NAME"
+    test_vars_overrides:
+      "location": '"us-central1"'
+      "serving_locality": '"GLOBAL_ACCESS"'
+    skip_test: true  # Can't establish a Github connection in automated tests.
```

Comment:
This should be `exclude_test: true` unless you expect to enable this in the near future